packages/blocks/src/api/raw-handling/figure-content-reducer.js

```diff
@@ -39,7 +38,13 @@ function isFigureContent( node, schema ) {
 function canHaveAnchor( node, schema ) {
 	const tag = node.nodeName.toLowerCase();
 
-	return has( schema, [ 'figure', 'children', 'a', 'children', tag ] );
+	return (
```

Comment:
This is a bit unwieldy. How about:

```js
return 'tag' in ( schema?.figure?.children?.a?.children ?? {} )
```

?

Reply:
Clever suggestion - thanks 👍 I like it. Going to use it for both the `has()` instances.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/43575/commits/a95ce146b3cd95141926dd41e9361656a17705d6